### PR TITLE
Faq: add LESS info

### PR DESCRIPTION
**Pull request by revolunet**
add pointers to have LESS support

### CODE REVIEW

File: docs/content/docs/faq/less.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: How to add LESS support
+---
+
+So if you want to add some global LESS to your phenomic (for components, you'd better use scoped CSS modules) :
+
+ - add a `layouts/LayoutContainer/index.global.less` file
```
Comment (thangngoc89) [inline docs/content/docs/faq/less.md]: Could you use uppercase for the first letter?

**Comment by thangngoc89** [comment]
We can create a new section in the docs and call it receipt. It's good for stuff like this.

And you should add using LESS in local scope.

**Comment by revolunet** [comment]
fixed :)

what do you mean by "using LESS in local scope." ?

**Comment by thangngoc89** [comment]
@revolunet I mean using LESS with css modules. (local-scoped)

**Comment by revolunet** [comment]
ah yes, i'd love a pointer to an example then :)

**Comment by thangngoc89** [comment]
Just duplicate these lines https://github.com/MoOx/phenomic/blob/2b44e19bfa095d2a4bb512c122dbb24380a83ef1/docs/webpack.config.babel.js#L53-L65

Replace postcss-loader with less-loader and modify the regex

**Comment by revolunet** [comment]
thanks :)

**Comment by MoOx** [comment]
I think I am going to explode the FAQ to move response in the appropriate part of the doc. I will still keep the FAQ list, but move response that can be. Will try to handle that.

**Comment by MoOx** [comment]
Going to make some adjustment & move that in the "Getting started" as it's a must have! Thank you! ❤️

**Comment by MoOx** [comment]
Merged by hand as there is conflict ;)

**Pull request closed without merging**